Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -94,24 +94,51 @@ def render_GET(self, request):
 
 class MyChannelRssFeedsEndpoint(MyChannelBaseEndpoint):
     """
-    Return the RSS feeds in your channel.
-
-    Example response:
-    {
-        "rssfeeds": [{
-            "url": "http://rssprovider.com/feed.xml",
-        }, ...]
-    }
+    This class is responsible for handling requests regarding rss feeds in your channel.
     """
 
-    def render_GET(self, request):
+    def get_my_channel_object(self):
         my_channel_id = self.channel_db_handler.getMyChannelId()
         channel_obj = self.session.lm.channel_manager.get_my_channel(my_channel_id)
         if my_channel_id is None or channel_obj is None:
```

Comment:
No need, just return the channel_obj then?

If bla is none:
return none
else:
return bla

is the same as return bla :)